src/components/NavBar/PlatformsMenu.tsx

```diff
@@ -1,33 +1,36 @@
-import './platforms.css';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { platforms } from '../../assets/platforms';
+import PlatformsStyle from './Platforms.module.css';
+import RoundButtonStyle from '../BasicComponents/RoundButton.module.css';
 
 interface PlatformsProps {
   switchPlatform: (newPlatform: string) => void;
 }
 
 const PlatformsMenu = ({ switchPlatform }: PlatformsProps) => (
-  <nav className="menu">
-    <input type="checkbox" className="menu-open" id="menu-open" />
-    <label className="menu-open-button" htmlFor="menu-open">
-      <span className="lines line-1" />
-      <span className="lines line-2" />
-      <span className="lines line-3" />
+  <nav className={PlatformsStyle.menu}>
+    <input type="checkbox" className={PlatformsStyle.menuOpen} id="menu-open" />
```

Comment:
not 100% sure - but it seems like you got the input and label section  and the platform section 

if they have two different logical reasons ( label for open menu and platform to  open different part in the page) you will be better diving them